packages/gatsby-source-shopify/src/type-builders/product-variant-type.ts

```diff
@@ -19,23 +19,20 @@ export function productVariantTypeBuilder(prefix: string): string {
         presentmentPrices: [${prefix}ProductVariantPricePair!]!
         price: Float!
         product: ${prefix}Product! @link(from: "_product", by: "id")
-        requiresShipping: Boolean! @deprecated(reason: "Use \`InventoryItem.requiresShipping\` instead.")
         selectedOptions: [${prefix}SelectedOption!]!
         sellingPlanGroupCount: Int! @proxy(from: "sellingPlanGroupsCount.count")
-        sku: String
         shopifyId: String!
+        sku: String
         storefrontId: String!
-        taxCode: String
         taxable: Boolean!
+        taxCode: String
         title: String!
         updatedAt: Date! @dateformat
-        weight: Float
-        weightUnit: ${prefix}WeightUnit!
```

Comment:
At the moment we remove these two fields, but if there's a way to proxy into `InventoryItem` we can avoid this.